src/styles/units/_mixins.scss

```diff
@@ -0,0 +1,10 @@
+@mixin makeCircle($size) {
```

Comment:
It is a good type of mixin, yet `radius` is making a circle. Add it here

Reply:
ADD